### ISSUE TITLE
Voyager runs into an error on start

### ISSUE THREAD

**Issue by ten-eugeniy** [cosmos/voyager#2143]
The error appears each time I try to run Voyager v2.0.8 on MacOS. Log is attached below. 

**WTR**
1. Run Voyager
2. Wait for **Sign in to Cosmos Voyager** form
3. Wait 5 seconds. You can click buttons, try to fill forms

```
Fri Mar 01 2019 10:57:14 GMT+0600 (+06) Running Cosmos-UI
main-process: initializing data directory (/Users/user/.cosmos-voyager/gaia-8001)
main-process: spawning /private/var/folders/70/_gjhtyqs5vgbtb1jp3q70ncm0000gn/T/AppTranslocation/5174D9F8-335F-4475-8DE7-E9D9C33B1DE7/d/Cosmos Voyager.app/Contents/Resources/app/bin/gaiacli with args ""version""
main-process: gaiacli: 0.24.2-1c38c704

main-process: gaiacli version: "0.24.2", expected: "0.24.1"
main-process: Querying node: 206.189.151.41
main-process: gaiacli exited with code 0
main-process: View has booted
main-process: Querying node: 77.74.192.243
main-process: Querying node: 103.72.145.36
main-process: Node 103.72.145.36 is alive.
main-process: Picked node: 103.72.145.36:26656
main-process: starting gaia rest server with nodeIP 103.72.145.36:26657
main-process: startLCD /Users/user/.cosmos-voyager/gaia-8001/lcd
main-process: spawning /private/var/folders/70/_gjhtyqs5vgbtb1jp3q70ncm0000gn/T/AppTranslocation/5174D9F8-335F-4475-8DE7-E9D9C33B1DE7/d/Cosmos Voyager.app/Contents/Resources/app/bin/gaiacli with args ""advanced" "rest-server" "--laddr" "tcp://localhost:9071" "--home" "/Users/user/.cosmos-voyager/gaia-8001/lcd" "--node" "103.72.145.36:26657" "--chain-id" "gaia-8001""
main-process: gaiacli: I[03-01|04:57:21.730] Starting RPC HTTP server on tcp://localhost:9071 module=rest-server 

main-process: gaiacli: I[03-01|04:57:21.731] REST server started                          module=rest-server 

main-process: gaiacli: I[03-01|04:57:22.711] Served RPC HTTP response                     module=rest-server method=GET url=/keys status=200 duration=0 remoteAddr=127.0.0.1:60816

main-process: gaia rest server ready
main-process: Signaling connected node
main-process: gaiacli: I[03-01|04:57:22.782] Served RPC HTTP response                     module=rest-server method=GET url=/keys status=200 duration=0 remoteAddr=127.0.0.1:60821

main-process: gaiacli: I[03-01|04:57:22.782] Served RPC HTTP response                     module=rest-server method=GET url=/keys status=200 duration=0 remoteAddr=127.0.0.1:60823

main-process: gaiacli: I[03-01|04:57:23.349] Served RPC HTTP response                     module=rest-server method=GET url=/node_version status=500 duration=634 remoteAddr=127.0.0.1:60817

main-process: Error in getting node SDK version, assuming node is incompatible. Error: Error: Request failed with status code 500
main-process: Querying node: 103.72.145.36
main-process: Node 103.72.145.36 is alive.
main-process: Picked node: 103.72.145.36:26656
main-process: starting gaia rest server with nodeIP 103.72.145.36:26657
main-process: startLCD /Usersuser.cosmos-voyager/gaia-8001/lcd
main-process: spawning /private/var/folders/70/_gjhtyqs5vgbtb1jp3q70ncm0000gn/T/AppTranslocation/5174D9F8-335F-4475-8DE7-E9D9C33B1DE7/d/Cosmos Voyager.app/Contents/Resources/app/bin/gaiacli with args ""advanced" "rest-server" "--laddr" "tcp://localhost:9071" "--home" "/Users/user/.cosmos-voyager/gaia-8001/lcd" "--node" "103.72.145.36:26657" "--chain-id" "gaia-8001""
main-process: gaiacli: I[03-01|04:57:24.107] Served RPC HTTP response                     module=rest-server method=GET url=/keys status=200 duration=0 remoteAddr=127.0.0.1:60825

main-process: gaia rest server ready
main-process: Signaling connected node
main-process: gaiacli: I[03-01|04:57:24.126] Served RPC HTTP response                     module=rest-server method=GET url=/keys status=200 duration=0 remoteAddr=127.0.0.1:60823

main-process: gaiacli: panic: resource temporarily unavailable

goroutine 1 [running]:

main-process: gaiacli: github.com/cosmos/cosmos-sdk/client/lcd.createHandler(0xc420147570, 0x5, 0x7ffeefbffc84)
	/go/src/github.com/cosmos/cosmos-sdk/client/lcd/root.go:78 +0x634
github.com/cosmos/cosmos-sdk/client/lcd.ServeCommand.func1(0xc420b3b440, 0xc420019080, 0x0, 0x8, 0x0, 0x0)
	/go/src/github.com/cosmos/cosmos-sdk/client/lcd/root.go:40 +0x8e
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).execute(0xc420b3b440, 0xc420018f80, 0x8, 0x8, 0xc420b3b440, 0xc420018f80)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:698 +0x46d
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x20dd820, 0x1a163f9, 0x24, 0xc420b1eee8)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:783 +0x2e4
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).Execute(0x20dd820, 0xc4205e1fc0, 0x20dd900)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:736 +0x2b
github.com/cosmos/cosmos-sdk/vendor/github.com/tendermint/tendermint/libs/cli.Executor.Execute(0x20dd820, 0x1b5ffe8, 0x2, 0xc420b29100)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/tendermint/tendermint/libs/cli/setup.go:89 +0x4e
main.main()
	/go/src/github.com/cosmos/cosmos-sdk/cmd/gaia/cmd/gaiacli/main.go:151 +0xcce
```

**Comment by faboweb** [comment]
Are you testing a released version? There actually is no Voyager v2.0.8. 🤷‍♂️But in generell the released binaries are heavily outdated. We will release a beta version of the new Voyager (on the web) somewhere very close (end of next weekish, please don't pin me down on if it's the week after :)).

**Comment by ten-eugeniy** [comment]
Hm.. That was unexpected. 
Thanks for the explanation.
